public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://www.pokt.network/wp-content/uploads/2020/06/cropped-pocket-icon-192x192.png" />
```

Comment:
At first I used a WordPress linked logo for the repositories, and then someone changed WordPress and all the images broke. We should come up with a better way to host this image - - - IPFS? Or an S3 bucket.

I'll give you a link to the logo on S3 shortly

Reply:
Thanks

Reply:
https://pocket-network-assets.s3-us-west-2.amazonaws.com/logos/cropped-pocket-icon-192x192.png